src/typescript/frontend/src/components/pages/home/components/main-card/MainCard.tsx

```diff
@@ -83,26 +84,19 @@ const MainCard = (props: MainCardProps) => {
             ref={globeImage}
             placeholder="empty"
           />
-
-          {[...new Intl.Segmenter().segment(featured?.market.symbolData.symbol ?? "🖤")].length ==
-          1 ? (
-            <div className="styled-emoji styled-single-emoji">
-              {featured?.market.symbolData.symbol ?? "🖤"}
-            </div>
-          ) : (
-            <div className="styled-emoji styled-double-emoji">
-              {featured?.market.symbolData.symbol}
-            </div>
-          )}
+          <Emoji
+            className={`styled-emoji ${featured?.market.emojis.length === 1 ? "styled-single-emoji" : "styled-double-emoji"}`}
+            emojis={featured?.market.emojis ?? emoji("black heart")}
+          />
         </Link>
 
         <Column maxWidth="100%" ellipsis>
-          <div className="flex flex-row">
-            <span className="pixel-heading-1 text-medium-gray pixel-heading-text">HOT</span>
+          <div className="flex flex-row content-center">
+            <span className="text-medium-gray pixel-heading-text">HOT</span>
             <span>&nbsp;</span>
-            <span className="pixel-heading-2 pixel-heading-text mt-[.2rem]">
-              {SYMBOL_EMOJI_DATA.byName("fire")?.emoji}
-            </span>
+            <div>
+              <Emoji className="pixel-heading-emoji" emojis={emoji("fire")} />
```

Comment:
Need to fix the vertical alignment on this one: 
<img width="1235" alt="Screenshot 2024-11-12 at 7 14 12 PM" src="https://github.com/user-attachments/assets/8f96c5a7-a721-4f0e-9dc9-7401fb52e461">

Reply:
Feel free to resolve after fixing